features/org.jboss.tools.arquillian.feature/feature.xml

```diff
@@ -14,7 +16,7 @@
       %copyright
    </copyright>
 
-   <license url="%licenseURL">
+   <license>
```

Comment:
No. the license.html is generated by the upstream license feature. I had previously thought we should remove this, but we in fact have to keep the license url="%licenseURL" part, so it points to license.html. The devdoc was updated to reflect this correction.

https://github.com/jbosstools/jbosstools-devdoc/commit/c3037f6632d550d834505822f912c4033408d205